skimage/future/graph/rag.py

```diff
@@ -431,28 +428,27 @@ def draw_rag(labels, rag, img, border_color=None, node_color='#ffff00',
     img : ndarray, shape (M, N, 3)
         Input image.
     border_color : colorspec, optional
-        Any matplotlib colorspec.
-    node_color : colorspec, optional
-        Any matplotlib colorspec. Yellow by default.
-    edge_color : colorspec, optional
-        Any matplotlib colorspec. Green by default.
-    colormap : colormap, optional
-        Any matplotlib colormap. If specified the edges are colormapped with
-        the specified color map.
+        Color with which the borders of regions are drawn.
+    edge_width : float, optional
+        The thickness with which the RAG edges are drawn.
+    edge_cmap : colormap, optional
+        Any matplotlib colormap with which the edges are drawn.
+    img_cmap : colormap, optional
+        Any matplotlib colormap with which the image is draw. If set to `None`
+        the image is drawn as it is.
     thresh : float, optional
-        Edges with weight below `thresh` are not drawn, or considered for color
+        Edges with weight above `thresh` are not drawn, or considered for color
```

Comment:
I'm not a fan of threshold in the first place. If someone wants to omit some edges they can always remove them from the graph.

And I don't think there is a right or wrong thing to do for any kind of graph, it really depends on what the user is trying to do.